gokart/mypy.py

```diff
@@ -413,6 +415,16 @@ def is_parameter_call(expr: Expression) -> bool:
 
     if isinstance(type_info, TypeInfo):
         return PARAMETER_FULLNAME_MATCHER.match(type_info.fullname) is not None
+
+    # Currently, luigi doesn't provide py.typed. it will be released next to 3.5.1.
+    # https://github.com/spotify/luigi/pull/3297
+    # With the following code, we can't assume correctly.
+    #
+    # from luigi import Parameter
+    # class MyTask(gokart.TaskOnKart):
+    #     param = Parameter()
+    if isinstance(type_info, Var) and luigi.__version__ <= '3.5.1':
```

Comment:
maybe?

```suggestion
    if isinstance(type_info, Var) and luigi.__version__ < '3.5.1':
```

Reply:
typed luigi is not yet released. So, I think it is correct.